### PR TITLE
fix:weight -10%

### PR DESCRIPTION
**Pull request by Amas9nine**
Issue #:
[Notion](https://trello.com/c/HrZyfSfd/13-discount-block-adaptive-layout)

## PR type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### after:
320px
100%weight
1280px
1024px
920px
768px
580px
480px
400px

## Does this introduce a breaking change?

- [ ] Yes (please describe):
- [x] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [X] Responsive layout and functionality have been tested
  - [X] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

File: src/components/qrcode/qcode.module.scss [hunk 1]
```diff
@@ -119,7 +119,7 @@
 }
 
 .aside {
-  margin: 0px auto 50px auto;
+  margin: 0px auto 70px auto;
```
Comment (Improove3567) [inline src/components/qrcode/qcode.module.scss:122]: у margin только 3 значения

File: src/components/qrcode/qcode.module.scss [hunk 2]
```diff
@@ -662,7 +663,7 @@
             width: 40px;
           }
 
-          span {
+          .title {
```
Comment (Improove3567) [inline src/components/qrcode/qcode.module.scss:666]: много похожих классов

File: public/shared/locales/ru/discountBlock.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "qrBlock": {
+    "bonus": "БОНУС",
+    "discount": "СКИДКА ",
+    "promocode": "Выберите ресторан из нашей побдорки хитов и получи промокод ",
+    "advantage": "ПРЕИМУЩЕСТВА"
+  },
+  "fastFeedback": "быстрая обратная связь",
```
Comment (asylbekovkairat) [inline public/shared/locales/ru/discountBlock.json:8]: все предложения должны начинаться с большое буквы

File: public/shared/locales/ru/discountBlock.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "qrBlock": {
+    "bonus": "БОНУС",
+    "discount": "СКИДКА ",
+    "promocode": "Выберите ресторан из нашей побдорки хитов и получи промокод ",
```
Comment (asylbekovkairat) [inline public/shared/locales/ru/discountBlock.json:5]: подборки*

File: public/shared/locales/ru/discountBlock.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    "advantage": "ПРЕИМУЩЕСТВА"
+  },
+  "fastFeedback": "быстрая обратная связь",
+  "checkedOrganization": "проверенные организации",
+  "famouseActors": "известные артисты",
+  "classificSpecialist": "классифицированные специалисты",
+  "quality": "качественное обслуживание",
+  "organization": "организация не выходя из дома"
```
Comment (asylbekovkairat) [inline public/shared/locales/ru/discountBlock.json:13]: с большой буквы

File: src/components/qrcode/QRcodeBlock.jsx [hunk 6]
```diff
@@ -57,28 +59,28 @@ function QRcodeBlock() {
...
             <div className={scss.text}>
               {card.map((item) => (
                 <div>
                   <img src={item.img} alt="img" />
-                  <span>{item.title}</span>
+                  <div className={scss.title}>{t(item.title)}</div>
                 </div>
               ))}
             </div>
             <div className={scss.textDropDown}>
```
Comment (asylbekovkairat) [inline src/components/qrcode/QRcodeBlock.jsx:79]: убрать второй мап, он вообще не нужен

Comment (Amas9nine) [reply src/components/qrcode/QRcodeBlock.jsx]: ok

**Review by asylbekovkairat — APPROVED** [review]

**Review by Improove3567 — APPROVED** [review]

**Pull request merged**